mlflow/utils/async_logging/async_logging_queue.py

```diff
@@ -42,7 +59,7 @@ def __init__(self, logging_func: callable([str, [Metric], [Param], [RunTag]])) -
         self._logging_func = logging_func
 
         self._stop_data_logging_thread_event = threading.Event()
-        self._is_activated = False
+        self._status = QueueStatus.TERMINATED
```

Comment:
Starting from terminated is a bit odd, can we add a new status `IDLE` for this purpose? Function-wise it should be the same, just TERMINATED at the very beginning looks odd..

Reply:
Agree that `TERMINEATED` doesn't sound the initial state. Will rename!